### ISSUE TITLE
[Feature Request] overview of online status of group participants

### ISSUE THREAD

**Issue by tooomm** [telegramdesktop/tdesktop#6223]
### Is your feature request related to a problem?
Telegram Desktop doesn't show the currently online members within a group.

### Additional context
On mobile the status information below the group name says for example `10 Members, 2 online`.
Within Telegram Desktop it only shows `10 Members`.

**Comment by SmartManoj** [comment]
In which version?

**Comment by tooomm** [comment]
I run 1.7.14 now. When I posted this it was 1.7.10 probably?
Sorry that my post missed that information.

**Comment by SmartManoj** [comment]
Is it happen  still ?

**Comment by 23rd** [comment]
For such a small number of participants, everything should be displayed normally.
![image](https://user-images.githubusercontent.com/4051126/61252081-0ccfef00-a765-11e9-83bf-431bb90764dc.png)

**Comment by tooomm** [comment]
>Is it happen  still ?

@SmartManoj yes. Version 1.7.14 still only displays „xx members“. You have to hover that phrase to see the list of participants with a „online“ or e.g. „last seen“ tag below their names.

**Comment by RiedleroD** [comment]
I have 1.9.21 and it gets displayed fine for me. This issue should probably be closed.

**Comment by RememberTheAir** [comment]
@RiedleroD it looks like it doesn't show for large groups, while mobile apps do. Tdesktop probably shows how many people are online based on the status of every single chat member when the chat is small, but the API offers a way to get the online members count for large groups too - which is not used by tdesktop yet, I suppose

**Comment by RiedleroD** [comment]
Oh, I see the problem. It's happening somewhere between 665 and 176 members. I'll look into it, maybe I can fix it and open a pull request.

**Comment by RememberTheAir** [comment]
@RiedleroD probably 200 is the magic number. You can look at the other clients' sourcecode to see how they behave

**Comment by RememberTheAir** [comment]
Still relevant

**Comment by RememberTheAir** [comment]
Still relevant
